libs/packages/components/src/lib/autocomplete-search/autocomplete-search.component.html

```diff
@@ -60,7 +98,47 @@
               [ngTemplateOutletContext]="{ $implicit: result }"
             >
             </ng-container>
-            <ng-container *ngIf="!itemTemplate">
+            <ng-container *ngIf="configuration.useCheckBoxes">
```

Comment:
Looks like this code will never be run. It's parent has an ngIf that includes "!configuration.useCheckBoxes" so if the parent is displayed, this ngIf will evaluate to false.

Reply:
yes, this part is not required, I updated it.